packages/shared/src/components/cards/v1/ArticlePostCard.tsx

```diff
@@ -116,7 +116,7 @@ export const ArticlePostCard = forwardRef(function PostCard(
                 </CardTitle>
 
                 {post.summary && (
-                  <CardTextContainer className="mt-4 text-theme-label-secondary">
```

Comment:
We should use our typo-body class I think, just because line-heights might not match. 
(actually noticed it's slightly different on webapp as well then, so better inline with design)

Reply:
TIL, thanks!

Reply:
Should be fixed now (although I didn't bother to change the screenshot, hope that's OK).